docs/README.md

```diff
@@ -737,6 +738,68 @@ Gets the latest tag and commit sha for the given pattern. The result is returned
           pattern: 1.0.0-alpha*
 ```
 
+### github-actions-ensure-sha-pinned-actions
```

Comment:
this is duplicating the docs inside the action?

Reply:
Yes in case people want to use this without the pre commit tooling boilerplate (?)